src/get-started/index.njk

```diff
@@ -0,0 +1,237 @@
+---
+title: Start designing services at HMRC 
+layout: oneColumnPage.njk
+---
+{% from "hmrc/components/page-heading/macro.njk"  import hmrcPageHeading %}
+
+<div class="govuk-grid-row">
+<div class="govuk-grid-column-two-thirds">
+
+{{ hmrcPageHeading({
+text: 'Start designing services at HMRC'
+}) }}
+
+<h2 class="govuk-heading-l" id="What-you-will-need">
+What you will need
+</h2>
+
+<p class="govuk-body">
+Designers need the right hardware, software and accounts to start designing HMRC services. Check that you have access to the following tools: 
+</p>
+
+<h3 class="govuk-heading-m" id="Hardware">
+  Hardware
+  </h3>
+
+  <ul class="govuk-list govuk-list--bullet">
+    <li>A computer that you have admin access to and can install software on</li>
+    </ul>
+
+<h3 class="govuk-heading-m" id="Software-systems-accounts">
+  Software/systems/accounts
+  </h3>
+
+  <ul class="govuk-list govuk-list--bullet">
+    <li>GOV.UK Prototype Toolkit</li>
+    <li>A code editor such as Visual Studio (VS) Code</li>
+    <li>Google Suite</li>
+    <li>Pseudo PID + Office 365</li>
+    <li>VPN access</li>
+    <li>GitHub</li>
+    </ul>
+
+<p class="govuk-body">
+If you need access to items on this list, speak to your manager/project lead, who will contact P&UCD Ops to set you up.
+</p>
+
+<h2 class="govuk-heading-l" id="Patterns-components-and-styles">
+  Patterns, components and styles
+</h2>
+
+<p class="govuk-body">
+All designs for HMRC services must follow <a href="https://www.gov.uk/guidance/government-design-principles" class="govuk-link">GOV.UK principles</a> and use <a href="https://design.tax.service.gov.uk/hmrc-design-patterns/" class="govuk-link">HMRC</a> and <a href="https://www.gov.uk/guidance/government-design-principles" class="govuk-link">GOV.UK</a>
+design patterns, components and styles. 
+</p>
+
+<p class="govuk-body">
+If a pattern exists in both HMRC and GOV.UK design libraries use the HMRC version, as it will have been created to meet specific needs and context within HMRC.
+</p>
+
+<p class="govuk-body">
+Designers should use these libraries for internal services where possible. When prototyping services for internal users, or services not on GOV.UK, designers should follow the relevant brand and typography guidance for their platform. 
+</p>
+
+<h3 class="govuk-heading-m" id="Get-support">
+  Get support
+</h3>
+
+<p class="govuk-body">
+Designers can get support with HMRC-specific patterns from the HMRC Design Resources team on HMRC Slack at <a href="https://hmrcdigital.slack.com/archives/C014S77JCJW?name=C014S77JCJW#" class="govuk-link">#team-design-system</a>. Service patterns also have a link to the discussions on <a href="https://github.com/hmrc/design-patterns/issues" class="govuk-link">GitHub</a>.
```

Comment:
These links don't make sense out of context. Could you change them to something like 'HMRC Slack at #team-design-system' & 'HMRC Github issues'?